react-frontend/src/components/Guides/CollaborationTools/collaborationTool.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+function CollaborationTool(props) {
+  return (
+    <div className="collaborationTool">
+      <a href={props.url}>
+        <strong>{props.name}</strong>
+      </a>{' '}
+      *
+      <ul>
+        <li>
+          <strong>Cost</strong>: {props.cost}
+        </li>
+        <li>
+          <strong>Platform</strong>: {props.platform}
+        </li>
+        <li>
+          <strong>Capacity</strong>: {props.capacity}
+        </li>
+        <li>
+          <strong>Features</strong>: {props.features}
+        </li>
+      </ul>
+    </div>
+  );
+}
+
+export default CollaborationTool;
```

Comment:
would be nice to see `PropTypes` for components like these

Reply:
agreed, I was being lazy